src/main/java/org/jabref/logic/autosaveandbackup/BackupManager.java

```diff
@@ -59,18 +59,19 @@ public class BackupManager {
     private final CoarseChangeFilter changeFilter;
     private final BibEntryTypesManager entryTypesManager;
 
-
     // Contains a list of all backup paths
     // During a write, the less recent backup file is deleted
     private final Queue<Path> backupFilesQueue = new LinkedBlockingQueue<>();
 
-    private boolean needsBackup = false;
+    private boolean needsBackup = true;
```

Comment:
Why is this boolean changed? This would revert https://github.com/JabRef/jabref/pull/9679 and thus probably reopen https://github.com/JabRef/jabref/issues/9488#issuecomment-1445149783.

Reply:
Oh good catch! Not intended

Reply:
Maybe, we should add a link to the PR and the issue there? - I also put `true` at the beginning to have the backup dir filled during development... -- I think, a short link at the comment is better than to say: "Look at git blame and read on there to understand this line of code ^^"